src/components/NavigationBar/NavigationBar.test.tsx

```diff
@@ -27,30 +69,97 @@ const navItems: NavigationItem[] = [
     id: "contact",
     label: "Contact",
     path: "/contact",
-    position: "left",
   },
+];
+
+const leftMenu = (navigationItems: NavigationItem[], onClick: (isOn: boolean) => void): React.ReactNode => {
+  return (
+    <div className="hidden lg:block md:ml-12">
+      <div className="flex h-full items-center">
+        {navigationItems.map((item, index) => {
+          return (
+            <div key={index} className="text-lg w-auto lg:ml-6">
+              <NavigationBarItem item={item} onClick={onClick} />
+            </div>
+          );
+          return "";
+        })}
+      </div>
+    </div>
+  );
+};
+
+const rightNavItems: NavigationItem[] = [
   {
     schema: NavigationItemType.IconButton,
     id: "settings",
     label: "Settings",
-    path: "/settings",
+    path: "settings",
     icon: Settings,
-    position: "right",
+  },
+  {
+    schema: NavigationItemType.LabelButton,
+    id: "create-documents",
+    label: "Create Doc",
+    path: "https://creator.tradetrust.io/",
+    className: "create-btn",
   },
   {
     schema: NavigationItemType.LabelButton,
     id: "verify",
     label: "Verify Doc",
     path: "/verify",
-    position: "right",
     className: "verify-btn",
   },
 ];
 
+const rightMenu = (navigationItems: NavigationItem[], onClick: (isOn: boolean) => void): React.ReactNode => {
+  return (
```

Comment:
interface this and ?onClick ? so u no need to pass empty function

Reply:
Changed, added pass in state instead of nulling onClick